presto-hive/pom.xml

```diff
@@ -493,6 +494,23 @@
                             <includes>
                                 <include>**/TestHivePushdownFilterQueries.java</include>
                                 <include>**/TestHivePushdownIntegrationSmokeTest.java</include>
+                                <include>**/TestHivePushdownDistributedQueries.java</include>
+                            </includes>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        <profile>
+            <id>test-hive-pushdown-filter-queries-advanced</id>
```

Comment:
@bhhari Why is it necessary to create this new job? I see TestHivePushdownDistributedQueries test included in both jobs: xxx-basic and xxx-advanced.

Reply:
had to create a new job because the 3 tests in 1 are timing out. I moved the distributed queries to a new one and the had a uncommited change in local. Update that now